plugins/modules/cohesity_storage_domain.py

```diff
@@ -0,0 +1,459 @@
+#!/usr/bin/python
+# Copyright (c) 2022 Cohesity Inc
+
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+# GNU General Public License v3.0+ (see https://www.gnu.org/licenses/gpl-3.0.txt)
+
+
+DOCUMENTATION = r"""
+---
+author: "Naveena (@naveena-maplelabs)"
+description:
+  - "Ansible Module used to register or remove the Cohesity Protection Sources to/from a Cohesity Cluster."
+  - "When executed in a playbook, the Cohesity Protection Source will be validated and the appropriate"
+  - "state action will be applied."
+module: cohesity_storage_domain
+options:
+  cluster:
+    aliases:
+      - cohesity_server
+    description:
+      - "IP or FQDN for the Cohesity Cluster"
+    type: str
+  cohesity_admin:
```

Comment:
This should just be cohesity_user ?

Reply:
We are using the same format in all the other modules. Can we change in all the modules?